rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-variable.adoc

```diff
@@ -0,0 +1,133 @@
+= Manage Global Variables
+
+Global variables store values that are the same in multiple processes and that frequently change. By using them, you can reduce the time involved and the error rate in the maintenance of the processes.
+
+Examples use cases of global variables are:
+
+* Tax rates (floating point value)
+* File paths (alphanumeric value)
+* Email addresses (alphanumeric value)
+* Approval (boolean value)
+* Offset for determining a screen coordinate (coordinate value)
+
+The RPA developer defines xref:rpa-builder::toolbox-variable-handling-activity-parameters.adoc[activity parameters for a process in RPA Builder]. 
+
+You create reusable global variables that process team members can link to an activity parameter in an automation to overwrite the initial value of the activity parameter:
+
+* xref:processautomation-deploy.adoc#test-configuration-link-globals[Linkable Activity Parameters in Test Configurations]
+* xref:processautomation-deploy.adoc#production-configuration-link-globals[Linkable Activity Parameters in Production Configurations]
+
+Use descriptive names for global variables and only use the variables in an appropriate context. If, for example, the number 42 happens to be both the number of employees in your company and the answer to a different question, then create two global variables (for example, `number_employees` and `answer`).
+
+Editing or deleting a global variable affects all configurations that use the variable.
+
+Check the usage of a variable before changing or deleting it.
+
+If you want to store login data securely, use xref:processautomation-prepare-deployment-credential.adoc[credentials].
+
+== Before You Begin
+
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
```

Comment:
```suggestion
* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the required privileges]:
```

Reply:
This seems familiar. Can you reuse any of this content?

Reply:
See previous comments, if applicable.

Reply:
line 30 processautomation-prepare-deployment-credential.adoc

Reply:
Anypoint Access Management will replace RPA Manager User Management in the next months. We will review the Before You Begin sections in the new IAM context. 
Each task needs a (slightly) different combination of permissions. Using a partial for one sentence might be a bit of overkill?